std/experimental/allocator/package.d

```diff
@@ -804,8 +804,7 @@ private enum hasPureDtor(T) = !hasElaborateDestructor!T ||
 private enum canSafelyDeallocPostRewind(T) = hasPurePostblit!T && hasPureDtor!T;
 
 /// Ditto
-T[] makeArray(T, Allocator)(auto ref Allocator alloc, size_t length,
-    auto ref T init)
+T[] makeArray(T, Allocator)(auto ref Allocator alloc, size_t length, T init)
```

Comment:
Why don't you use `in` or `inout` here? Passing parameters by reference if possible is usually faster.

Reply:
@wilzbach which parameter do you mean?

Reply:
@andralex my guess would be `T init` -> `in T init`

Reply:
`in`/`inout` don't pass by reference, and the previous code with ref didn't work. I'll pull this as is, improvements are welcome